bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/InsteonBindingConstants.java

```diff
@@ -46,6 +47,7 @@
     public static final String BROADCAST_ON_OFF = "broadcastOnOff";
     public static final String CONTACT = "contact";
     public static final String COOL_SET_POINT = "coolSetPoint";
+    public static final String DAWNDUSK = "dawndusk";
```

Comment:
Can we use something more generic than "dawndusk"? This assumes that the variable is associated with the sun rise/set.

Reply:
I can make it whatever you'd like, but I think this is the traditional way to refer to this feature on motion sensors going back to old X10 stuff. How about lightdark?

Reply:
change to:
public static final String LIGHTDARK = "lightdark";